docs/user_guide/Introduction_to_6.4.0.rst

```diff
@@ -0,0 +1,175 @@
+
+Introduction to SIMP 6.4.0
+==========================
+
+
+High-level workflow
+
+**(Admin) installs SIMP from ISO**
+
+.. image:: ./20190319_iso_critical_path-40000_ft_overview_A.png
+   :alt: A diagram that describes an Admin installing SIMP from an ISO Info
+
+(Admin) installs SIMP from ISO, using the SIMP docs procedure, `ISO Installation (preferred) <https://simp.readthedocs.io/en/6.3.3/getting_started_guide/Installation_Options/ISO/index.html>`_
+
+* The OS and RPMs are installed according to the ISO's `auto.cfg. <https://github.com/simp/simp-core/blob/master/build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg>`_
```

Comment:
Discussion of auto.cfg is a detail for developers only.  I would remove this detail.